src/afl-sharedmem.c

```diff
@@ -239,6 +264,37 @@ u8 *afl_shm_init(sharedmem_t *shm, size_t map_size,
 
   }
 
+  snprintf(shm->var_bytes_g_shm_file_path, L_tmpnam, "/afl_var_bytes_%d_%ld",
+           getpid(), random());
+
+  /* create the shared memory segment as if it was a file */
+  shm->var_bytes_g_shm_fd =
+      shm_open(shm->var_bytes_g_shm_file_path, O_CREAT | O_RDWR | O_EXCL,
```

Comment:
Just a nitpick: the original idea behind the `afl_shm` functions was to create one `sharedmem_t` per shared map.
That way we don't have to call `shm_open`(or whatever the names are on other operating systems), but it "just works".